src/Applications/beta/examples/Get-MgBetaApplicationExtensionProperty.md

```diff
@@ -1,14 +1,17 @@
-### Example 1: Using the Get-MgBetaApplicationExtensionProperty Cmdlet
+### Example 1: {{ Add title here }}
```

Comment:
The PR is removing existing cmdlet examples. Please review the example update pipeline.

Reply:
@peombwa thanks for flagging this. I have created another PR to resolve this issue.